pkg/chartutil/coalesce.go

```diff
@@ -285,9 +286,49 @@ func coalesceTablesFullKey(printf printFn, dst, src map[string]interface{}, pref
 			} else {
 				printf("warning: cannot overwrite table with non table for %s (%v)", fullkey, val)
 			}
+		} else if islist(val) {
```

Comment:
`islist` and `coalesceListsFullKey` don't work when the list is multidimensional i.e. `[][]interface{}` which i'm not 100% sure if Helm supports that kind of value anyway so it might not be necessary

Reply:
I don't know such cases